www/docs/en/dev/guide/platforms/win8/index.md

```diff
@@ -287,6 +287,37 @@ powershell -Command " & {dir -path cert:\LocalMachine\My | where { $_.Subject -l
 
 Once these final values are provided. Cordova should successfully package and sign the app.
 
+## MSBuild build flags
+
+Similar to other platforms ([`--gradleArg` on Android](../android/index.html#setting-gradle-properties), [`--buildFlag` on iOS](../ios/index.html#xcode-build-flags)) you can pass custom flags to MSBuild. To do this you have two options:
+
+- add one or more `--buildFlag` options to `cordova build windows` or `cordova run windows` commands:
+
+      ```
+      cordova build windows -- --buildFlag /clp:Verbosity=normal --buildFlag /p:myCustomProperty=Value
```

Comment:
Do we support both `--buildFlag="/clp:Verbosity=normal"` and `--buildFlag /clp:Verbosity=normal` notations?
Maybe a nit but Android and iOS platform guides use `=` notation.